pkg/amber/provenance.go

```diff
@@ -94,6 +95,20 @@ func (p *ValidatedProvenance) GetBuildCmd() ([]string, error) {
 	return buildConfig.Command, nil
 }
 
+// Guess the digest for the Builder Image.
```

Comment:
Is this really a `guess`? 

The comments for exported functions should start with the function name `GetBuilderImageDigest`.

Reply:
Thanks! Done!